.circleci/config.yml

```
@@ -0,0 +1,43 @@
+version: 2.1
+
+commands:
+  run-checks:
+    description: "Run basic checks"
+    steps:
+      - checkout
+      - run: npm ci
+      - run: npm run lint
+  run-build:
+    description: "Ensure compiled assets are up to date and HTML is valid"
```

```suggestion
    description: "Ensure compiled assets are up to date"
```

This doesn't look like it verifies valid HTML.